### DISCUSSION TITLE
Edit existing & create new measures in PowerBI Desktop

### DISCUSSION THREAD

**Discussion by winglessbuzzard** [DaxStudio/DaxStudio#1061] [Q&A] [answered]
I have a PowerBI pibx file. I've connected it to DAX Studio and can see & interact with my stuff as expected (according to videos & other instructions I've found). One feature that I assume should exist in DAX studio that I can't seem to find is: directly updating the DAX my PBIX from DAX Studio into PowerBI.
Is the current workflow(?):
Create/tweak DAX in DAX studio
copy it in DAX studio (as text)
paste it into the proper place in PowerBI

Is there some sort of save/push/etc button or function I'm missing that I can execute to, say, push the new definition of a measure from DAX Studio into the PBIX file? If not, I'd super want this feature.

**Comment by marcosqlbi** [comment, marked as answer]
You should use Tabular Editor for that - there is a [free, open-source version (v2)](https://github.com/TabularEditor/TabularEditor/releases/latest) and a [commercial version (v3)](https://www.tabulareditor.com/).

**Reply by winglessbuzzard** [reply]
Got it. Thanks!
